_docs/release-notes.md

```diff
@@ -360,6 +378,115 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
+
+The manual bolus history? We're getting all of that.  
+[Feature Requirements](https://trello.com/c/asHiqHmq/18-502-medtronic-6-history-message-manual-bolus-ux0-dev3-tst3)
+
+Calculated bolus history, too.  
+[Feature Requirements](https://trello.com/c/pdOnbBwT/20-501-medtronic-6-history-message-calculator-bolus-ux0-dev5-tst3)
+
+And all those recorded and reported blood glucose readings? Yup.  
+[Feature Requirements](https://trello.com/c/BHVmZoHh/19-500-medtronic-6-history-message-smbg-values-ux0-dev3-tst3)
+
+Can't forget about reservoir changes and prime event history, can we?  
+[Feature Requirements](https://trello.com/c/YwljcNPd/21-uploader-6-series-rewind-prime-device-events)
+
+CGM data? You're coming too.  
+[Feature Requirements](https://trello.com/c/HNi2st5n/29-497-medtronic-6-history-message-cgm-data-ux0-dev3-tst3)
+
+No matter what time zone you live in, our UTC bootstrapping algorithm will ensure your data is in sync.  
+[Feature Requirements](https://trello.com/c/vwaKLzuT/17-test-medtronic-6-utc-bootstrapping)
+
+Our time check feature also helps with that.  
+[Feature Requirements](https://trello.com/c/0bEUjkmz/25-xs-new-card-6-series-time-check)
+
+New images and UI components were added to the Uploader for a paired and unpaired Contour Next Link 2.4 meter.  
+[Feature Requirements](https://trello.com/c/9LBpSPxH/4-new-images-for-6-series-ui)
+
+This includes a proper option to select in the "Choose Devices" section.  
+[Feature Requirements](https://trello.com/c/BF1URAvH/31-494-s-uploader-medtronic-6-driver-copy-for-choose-devices-screen)
+
+All of those data points need to be displayed correctly on the Basics view.  
+[Feature Requirements](https://trello.com/c/JgH7y7VY/26-6-series-basics-page)
+
+Device settings also are properly displayed.  
+[Feature Requirements](https://trello.com/c/JkZGdYaS/28-6-series-device-settings)
+
+Including all the different basal schedule profiles.  
+[Feature Requirements](https://trello.com/c/CwJHioPO/32-504-medtronic-6-1b-implement-basal-profiles-ux0-dev3-tst1)
+
+Over the course of internal and beta testing we documented and resolved a handful of bugs. Ready for those?  
+
+We documented and resolved a bug that showed a previously scheduled basal extending into Auto Mode time if Auto Mode started with no insulin delivery.  
+[Feature Requirements](https://trello.com/c/sYxXIylr/3-if-auto-mode-starts-with-no-microbolus-delivery-data-is-incorrect)
+
+We also documented and resolved a bug that displayed temp basals during an upload as a scheduled basal.  
+[Feature Requirements](https://trello.com/c/fk1vqCBz/6-uploader-600-series-when-a-temp-basal-starts-before-an-upload-data-is-uploaded-incorrectly)
+
+We documented and resolved a bug that displayed basal schedule changes during a temp basal period incorrectly.  
+[Feature Requirements](https://trello.com/c/WZbBxkXA/7-670g-add-raw-microbolus-data-to-payload-field-of-basal-data-type)
```

Comment:
Not sure if this is the correct card for this bug description? (The same card is mentioned earlier)